--- origsrc/njl-plugins/Makefile	2005-02-27 18:24:58.000000000 -0600
+++ src/njl-plugins/Makefile	2009-09-10 22:58:46.352343700 -0500
@@ -11,13 +11,13 @@ clean:
 	rm $(ALL)
 
 noise_1921.so: noise_1921.c
-	gcc -Wall -shared -Xlinker -Bdynamic -O2 -fPIC -o noise_1921.so noise_1921.c -lm
+	$(CC) -Wall -shared -Xlinker -Bdynamic -O2 -o noise_1921.so noise_1921.c -lm
 
 noise_1922.so: noise_1922.c
-	gcc -Wall -shared -Xlinker -Bdynamic -O2 -fPIC -o noise_1922.so noise_1922.c -lm
+	$(CC) -Wall -shared -Xlinker -Bdynamic -O2 -o noise_1922.so noise_1922.c -lm
 
 eir_1923.so: eir_1923.c
-	gcc -Wall -shared -Xlinker -Bdynamic -O2 -fPIC -o eir_1923.so eir_1923.c -lm
+	$(CC) -Wall -shared -Xlinker -Bdynamic -O2 -o eir_1923.so eir_1923.c -lm
 
 risset_1924.so: risset_1924.c
-	gcc -Wall -shared -Xlinker -Bdynamic -O2 -fPIC -o risset_1924.so risset_1924.c -lm
+	$(CC) -Wall -shared -Xlinker -Bdynamic -O2 -o risset_1924.so risset_1924.c -lm
